docs/articles/Core/Serialization/EntryFormat.md

```diff
@@ -54,33 +54,109 @@ The recursive `SubEntries` structure represents properties of a `EntryValueType.
 Example class:
 
 ````cs
-public class Root
+public class PackagingResource
```

Comment:
I like the new example a lot! Could you please just change the property names to the c#conventions, i.e. PascalCase.
You could also remove the DataMember attributes as they are relevant only for storing the properties to the data base which is not necessarily part of these docs 😊